ovh/types_order_cart.go

```diff
@@ -122,3 +122,16 @@ func (opts *OrderCartItemCreateOpts) FromResource(d *schema.ResourceData) *Order
 
 	return opts
 }
+
+type OrderCartCheckout struct {
+	OrderID int64                   `json:"orderId"`
+	Prices  OrderCartCheckoutPrices `json:"prices"`
```

Comment:
why don't you just add the field `Prices` in existing struct `MeOrder` ?

Reply:
the APIv6 returned type is different from /checkout and /me/order
that's not the same output